### PR TITLE
Join translations from Crowdin to one file

### PR DESCRIPTION
**Pull request by cat2608**

### CODE REVIEW

File: crowdin.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+"project_identifier" : ""
+"api_key" : ""
+"base_path" : "src/assets/i18n"
+#"base_url" : ""
```
Comment (daves125125) [inline crowdin.yml]: commented base_url param?

File: crowdin.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+
+"project_identifier" : ""
+"api_key" : ""
```
Comment (daves125125) [inline crowdin.yml]: Do we need to specify these params as empty placeholders or can we eliminate them from the yml and just use ENV Vars etc?

Comment (cat2608) [inline crowdin.yml]: @daves125125 I think I need to write it like this: 
https://github.com/transferwise/blog/blob/master/crowdin.yaml
What do you think?

Comment (lonamiaec) [inline crowdin.yml]: I think you need to specify the environment variable name expected here

**Pull request merged**